client/react/src/app.tsx

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 import {Replicache} from 'replicache';
 import {useSubscribe} from 'replicache-react';
 
-import {M} from '../../shared/mutators';
-import {listTodos, TodoUpdate} from '../../shared/todo';
+import {M} from 'replicache-quickstarts-shared';
```

Comment:
The import lines can be combined, probably true in a number of other files as well.